src/storage.js

```diff
@@ -0,0 +1,26 @@
+const redis = require('./lib/redis');
+
+// Redis Keys
+const FEED_ID = 'feed_id';
+const FEEDS = 'feeds';
+
+module.exports = {
+  addFeed: async (name, url) => {
+    // "If the key does not exist, it is set to 0 before performing the operation"
+    // https://redis.io/commands/INCR
+    const feedId = await redis.incr(FEED_ID);
+    await redis
+      .multi()
+      // Using hmset() until hset() fully supports multiple fields:
+      // https://github.com/stipsan/ioredis-mock/issues/345
+      // https://github.com/luin/ioredis/issues/551
+      .hmset(feedId, 'name', name, 'url', url)
+      .sadd('feeds', feedId)
```

Comment:
`'feeds'` should be `FEEDS` here.

Reply:
This got resolved in GitHub, but the code is still wrong.  Prefer the const `FEEDS` as the first argument here.